### ISSUE TITLE
About transfer learning

### ISSUE THREAD

**Issue by FeiMiBa** [deepinsight/insightface#1247]
Hi, I followed your instruction to train a model with MS1M training set, and I ended up achieving 99+@lfw, 95+@cfp, 95+@AgeDB. This pretrained model also achieved 97+ on my own testing set.

Then I used this pretrained model of mine to do transfer learning on my own training set, which is quite different from MS1M/LFW/CFP/AgeDB.... When I finished the tranfer learning, the final model achieved 99+ on my own testing set, but only 80+@lfw, 70+@cfp, 70+@AgeDB.

I wonder if I made any mistakes? Is there any ways to imporve the performance on my own testing set while remaining fair enough good performance on other testing sets?

Thank you for your time.

**Comment by nttstar** [comment]
You can do L2 distance loss mimic.

**Comment by saadnajib** [comment]
FeimiBa can i know how did you perform transfer learning ? I tried training on my own custom dataset using cosfacer50 pretrained model, after setting resume=true but on testing accuracy decreased on my custom data test set. can you guide your method to transfer learning.
 Thank you for your time.

**Comment by sriram-dsl** [comment]
@FeiMiBa can i know how did you perform tranfer learning

**Comment by azadehashouri** [comment]
@FeiMiBa 
Hi FeiMiBa.

I want to use transfer learning to train ArcFace Torch on my custom dataset.

My custom dataset contains 2 picture per person.

Could you help me how should I perform transfer learning?

Thanks in advance

**Comment by acel122** [comment]
@azadehashouri hello! did you already know how to do transfer learning on the custom dataset? i encountered the similar problem too, thank you
